### DISCUSSION TITLE
Disable new comments without locking issue

### DISCUSSION THREAD

**Discussion by brshallo** [community/community#4850] [Projects and Issues]
Is there a way to disable all new comments from non-collaborators on an issue while not completely locking jt?

Specifically, making it so that comments can no longer be added on the issue directly but references from another issue or PR will still show-up (even after the issue has been "locked")?

**Comment by vbrozik** [comment]
It also seems that locking an issue also locks voting on it. I think that this side-effect is often undesirable. Feedback on importance of an issue is almost always very useful.

**Comment by Mahmoud2882006** [comment]
What is your name?

**Comment by keunes** [comment]
Seems I created the same request, but kinda inverted wording: [lock conversations while still allowing reactions/votes](https://github.com/orgs/community/discussions/36107).
